pkg/cmd/opts/install.go

```diff
@@ -777,10 +777,39 @@ func (o *CommonOptions) InstallMinishift() error {
 
 // InstallGcloud installs gcloud cli
 func (o *CommonOptions) InstallGcloud() error {
-	if runtime.GOOS != "darwin" || o.NoBrew {
-		return errors.New("please install missing gcloud sdk - see https://cloud.google.com/sdk/downloads#interactive")
+	versionResolver, err := o.GetVersionResolver()
+	if err != nil {
+		return err
+	}
+	stableVersion, err := versionResolver.StableVersionNumber(versionstream.KindPackage, "gcloud")
+	if err != nil {
+		return err
 	}
-	return o.RunCommand("brew", "cask", "install", "google-cloud-sdk")
+	architecture := runtime.GOARCH
+	if runtime.GOARCH == "amd64" {
```

Comment:
do any other GOARCH strings need mapping? I guess that's the one we really need.

Reply:
According to what is available on the gcloud sdk website an amd32 might need a different mapping if that architecture needs downloading.
More specifically this link : `https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-282.0.0-linux-x86.tar.gz`